sdc/__init__.py

```diff
@@ -60,10 +60,9 @@
     TODO: Needs to detect 'import Pandas' and align initialization according to it
     """
 
-    # sdc.config.numba_compiler_define_nopython_pipeline_orig = \
-    #     numba.compiler.DefaultPassBuilder.define_nopython_pipeline
-    # numba.compiler.DefaultPassBuilder.define_nopython_pipeline = \
-    #     sdc.datatypes.hpat_pandas_dataframe_pass.sdc_nopython_pipeline_lite_register
+    sdc.config.numba_compiler_define_nopython_pipeline_orig = numba.compiler.DefaultPassBuilder.define_nopython_pipeline
```

Comment:
I believe since @AlexanderKalistratov PR with rewrites this should be commented out. Please pay attention while rebasing.